### PR TITLE
FAT-4542/C359015

### PR DESCRIPTION
**Pull request by Sherzod-Kenjaev**
https://issues.folio.org/browse/FAT-4542
https://foliotest.testrail.io/index.php?/cases/view/359015

### CODE REVIEW

File: cypress/support/fragments/inventory/inventoryInstance.js [hunk 1]
```diff
@@ -162,37 +188,167 @@ export default {
...
   editMarcBibliographicRecord:() => {
     cy.do(actionsButton.click());
     cy.do(editMARCBibRecordButton.click());
     cy.expect(Pane({ id: 'quick-marc-editor-pane' }).exists());
   },
 
+  importInstance() {
+    cy.do(Section({ id: 'pane-results' }).find(Button('Actions')).click());
+    cy.do(Section({ id: 'actions-menu-section' }).find(Button({ id: 'dropdown-clickable-import-record' })).click());
+    cy.expect(Modal({id: 'import-record-modal'}).exists());
```
Comment (IhorBohdan) [inline cypress/support/fragments/inventory/inventoryInstance.js:232]: Move to const

File: cypress/support/fragments/inventory/inventoryInstance.js [hunk 2]
```diff
@@ -162,37 +188,167 @@ export default {
...
+      searchOptionBtn.exists(),
+      browseOptionBtn.exists(),
+      searchTextArea.exists(),
+      searchButtonDisabled.exists(),
+      resetAllButton.exists(),
+      sourceFileAccordion.find(MultiSelect({ label: including('Authority source') })).exists(),
+      sourceFileAccordion.find(MultiSelect({ selected: including('LC Name Authority file (LCNAF)') })).exists(),
+      subjectHeadingAccordion.find(Button('Thesaurus')).has({ ariaExpanded: 'false' }),
+      headingTypeAccordion.find(Button('Type of heading')).has({ ariaExpanded: 'false' }),
+      createdDateAccordion.find(Button('Date created')).has({ ariaExpanded: 'false' }),
```
Comment (IhorBohdan) [inline cypress/support/fragments/inventory/inventoryInstance.js:263]: move to const

File: cypress/support/fragments/inventory/inventoryInstance.js [hunk 3]
```diff
@@ -162,37 +188,167 @@ export default {
...
+      MultiColumnListRow({index: 0}).find(Button({ ariaLabel: 'Link' })).exists(),
+      MultiColumnListCell({row: 0, innerHTML: including('<b>Authorized</b>') }).exists(),
+      Button({ id: 'authority-result-list-prev-paging-button', disabled: true }).exists(),
+      Button({ id: 'authority-result-list-next-paging-button', disabled: true }).exists(),
+    ]);
+    cy.expect([
+      Button({ id: 'authority-result-list-prev-paging-button', disabled: true }).exists(),
+      or(
+        Button({ id: 'authority-result-list-next-paging-button', disabled: true }).exists(),
+        Button({ id: 'authority-result-list-next-paging-button', disabled: false }).exists(),
```
Comment (IhorBohdan) [inline cypress/support/fragments/inventory/inventoryInstance.js:322]: move to const

File: cypress/support/fragments/inventory/inventoryInstance.js [hunk 4]
```diff
@@ -162,37 +188,167 @@ export default {
...
+
+  checkSearchResultsTable() {
+    cy.expect([
+      MultiColumnListHeader({ id: 'list-column-link' }).has({ content: 'Link' }),
+      MultiColumnListHeader({ id: 'list-column-authreftype' }).has({ content: 'Authorized' }),
+      MultiColumnListHeader({ id: 'list-column-headingref' }).has({ content: 'Heading/Reference' }),
+      MultiColumnListHeader({ id: 'list-column-headingtype' }).has({ content: 'Type of heading' }),
+      MultiColumnListRow({index: 0}).find(Button({ ariaLabel: 'Link' })).exists(),
+      MultiColumnListCell({row: 0, innerHTML: including('<b>Authorized</b>') }).exists(),
+      Button({ id: 'authority-result-list-prev-paging-button', disabled: true }).exists(),
```
Comment (IhorBohdan) [inline cypress/support/fragments/inventory/inventoryInstance.js:315]: move to const

File: cypress/support/fragments/inventory/inventoryInstance.js [hunk 5]
```diff
@@ -162,37 +188,167 @@ export default {
...
   },
+
   editMarcBibliographicRecord:() => {
     cy.do(actionsButton.click());
     cy.do(editMARCBibRecordButton.click());
     cy.expect(Pane({ id: 'quick-marc-editor-pane' }).exists());
   },
 
+  importInstance() {
+    cy.do(Section({ id: 'pane-results' }).find(Button('Actions')).click());
```
Comment (IhorBohdan) [inline cypress/support/fragments/inventory/inventoryInstance.js:230]: move Butoon to const

File: cypress/support/fragments/inventory/inventoryInstance.js (hunk 1 above)
Comment (Sherzod-Kenjaev) [reply cypress/support/fragments/inventory/inventoryInstance.js]: Done

File: cypress/support/fragments/inventory/inventoryInstance.js (hunk 2 above)
Comment (Sherzod-Kenjaev) [reply cypress/support/fragments/inventory/inventoryInstance.js:283]: Done

File: cypress/support/fragments/inventory/inventoryInstance.js (hunk 3 above)
Comment (Sherzod-Kenjaev) [reply cypress/support/fragments/inventory/inventoryInstance.js]: Done

File: cypress/support/fragments/inventory/inventoryInstance.js (hunk 4 above)
Comment (Sherzod-Kenjaev) [reply cypress/support/fragments/inventory/inventoryInstance.js]: Done

File: cypress/support/fragments/inventory/inventoryInstance.js (hunk 5 above)
Comment (Sherzod-Kenjaev) [reply cypress/support/fragments/inventory/inventoryInstance.js]: Done

**Review by MaksimTerekhov — APPROVED** [review]

**Review by nayimovag — APPROVED** [review]

**Pull request merged**